tensorflow/python/compiler/tensorrt/test/BUILD

```diff
@@ -63,566 +59,59 @@ filegroup(
-    ],
-    python_version = "PY3",
-    tags = [
-        "no_cuda_on_cpu_tap",
-        "no_rocm",
-        "no_windows",
-        "nomac",
-    ],
-    xla_enable_strict_auto_jit = False,
-    deps = [
-        "//tensorflow/python:client_testlib",
-        "//tensorflow/python:framework_test_lib",
-        "//tensorflow/python/compiler/tensorrt:tf_trt_integration_test_base",
-    ],
-)
-
-cuda_py_test(
-    name = "quantization_test",
-    srcs = [
-        "quantization_test.py",
-    ],
-    python_version = "PY3",
-    tags = [
-        "no_cuda_on_cpu_tap",
-        "no_rocm",
-        "no_windows",
-        "nomac",
-    ],
-    xla_enable_strict_auto_jit = False,
-    deps = [
-        "//tensorflow/python:client_testlib",
-        "//tensorflow/python:framework_test_lib",
-        "//tensorflow/python/compiler/tensorrt:tf_trt_integration_test_base",
-    ],
-)
-
-cuda_py_test(
-    name = "rank_two_test",
-    srcs = [
-        "rank_two_test.py",
-    ],
-    python_version = "PY3",
-    tags = [
-        "no_cuda_on_cpu_tap",
-        "no_rocm",
-        "no_windows",
-        "nomac",
-    ],
-    xla_enable_strict_auto_jit = False,
-    deps = [
-        "//tensorflow/python:client_testlib",
-        "//tensorflow/python:framework_test_lib",
-        "//tensorflow/python/compiler/tensorrt:tf_trt_integration_test_base",
-    ],
-)
-
-cuda_py_test(
-    name = "reshape_transpose_test",
-    srcs = [
-        "reshape_transpose_test.py",
-    ],
-    python_version = "PY3",
-    tags = [
-        "no_cuda_on_cpu_tap",
-        "no_rocm",
-        "no_windows",
-        "nomac",
-    ],
-    xla_enable_strict_auto_jit = False,
-    deps = [
-        "//tensorflow/python:client_testlib",
-        "//tensorflow/python:framework_test_lib",
-        "//tensorflow/python/compiler/tensorrt:tf_trt_integration_test_base",
-    ],
-)
-
-cuda_py_test(
-    name = "tf_function_test",
-    srcs = [
-        "tf_function_test.py",
-    ],
-    python_version = "PY3",
-    tags = [
+oss_tests = [
+    "batch_matmul_test",
+    "biasadd_matmul_test",
+    "binary_tensor_weight_broadcast_test",
+    "cast_test",
+    "concatenation_test",
+    "const_broadcast_test",
+    "dynamic_input_shapes_test",
+    "identity_output_test",
+    "int32_test",
+    "lru_cache_test",
+    "memory_alignment_test",
+    "multi_connection_neighbor_engine_test",
+    "neighboring_engine_test",
+    "quantization_test",
+    "rank_two_test",
+    "reshape_transpose_test",
+    "topk_test",
+    "trt_engine_op_shape_test",
+    "trt_mode_test",
+    "unary_test",
+    "vgg_block_nchw_test",
+    "vgg_block_test",
+]
+
+no_oss_tests = [
+    "base_test",
+    "conv2d_test" # "conv2d_test.py",  # b/198501457
+]
+# "base_test.py", # TODO(b/165611343): Need to address the failures for CUDA 11 in OSS build.
+
+base_tags = [
+    "no_cuda_on_cpu_tap",
+    "no_rocm",
+    "no_windows",
+    "nomac",
+]
+
+tensorrt_py_test(oss_tests, base_tags)
```

Comment:
I got this error:
All calls to rules or macros should pass arguments by keyword (arg_name=value) syntax.

Please fix here and other similar places.

Reply:
FIXED